abstract.md

```diff
@@ -1,3 +1,5 @@
-This document provides guidance on evaluating how well websites conform to the [Web Content Accessibility Guidelines (WCAG) 2](https://www.w3.org/WAI/standards-guidelines/wcag/). It describes a procedure to evaluate websites and includes considerations to guide evaluators and to promote good practice. It does not provide instructions for evaluating web content feature by feature, which is addressed by WCAG 2 success criteria. This document is one of a series of informative W3C/WAI resources about [Evaluating Websites for Accessibility](https://www.w3.org/WAI/test-evaluate/) that complement the [WCAG 2 Documents](https://www.w3.org/WAI/standards-guidelines/wcag/docs/). It does not define additional WCAG 2 requirements nor does it replace or supersede them in any way.
+This document describes a procedure to evaluate how well digital services and products conform to the [Web Content Accessibility Guidelines (WCAG) 2](https://www.w3.org/WAI/standards-guidelines/wcag/). 
 
-The methodology described in this document is intended for people who are experienced in evaluating accessibility using WCAG 2 and its supporting resources. It provides guidance on good practice in defining the evaluation scope, exploring the target website, selecting representative samples from websites where it is not feasible to evaluate all content, auditing the selected samples, and reporting the evaluation findings. It is primarily designed for evaluating existing websites, for example, to learn about them and to monitor their level of accessibility. It can also be useful during earlier design and development stages of websites. It applies to static and dynamically generated websites, mobile websites and applications, and other types of websites. It does not specify particular web technologies, evaluation tools, web browsers, assistive technologies, or other software to use for evaluation. It is suitable for use in different evaluation contexts, including self-assessment and third-party evaluation.
\ No newline at end of file
+It provides technology-agnostic guidance on defining the evaluation scope, exploring the target product, selecting representative samples from websites, auditing the selected samples, and reporting the evaluation findings. It is suitable for use in different evaluation contexts, including self-assessment and third-party evaluation. 
```

Comment:
should this 

> selecting representative samples from websites

use some other term than "websites"?

Reply:
updated to say 'product'